etcdctl/ctlv3/command/del_command.go

```diff
@@ -70,6 +74,10 @@ func getDelOp(args []string) (string, []clientv3.OpOption) {
 			cobrautl.ExitWithError(cobrautl.ExitBadArgs, fmt.Errorf("too many arguments, only accept one argument when `--prefix` or `--from-key` is set"))
 		}
 		opts = append(opts, clientv3.WithRange(args[1]))
+		if !delRange {
+			fmt.Fprintf(os.Stderr, "Warning: This operations is implicitly deleting range of keys. Please add flag `--range` to make it explicit. Will sleep in v3.6.")
```

Comment:
Some grammatical errors + kind of ambiguous info because it doesn't specify that the command *could be wrong*. Consider changing to "Warning: This operation will implicitly delete a range of keys. Please add flag `--range` to make it explicit or check your command input."

Reply:
OK. Fix. 
MeanWhile. Please take a look [this](https://github.com/etcd-io/etcd/pull/13748#discussion_r817905830)